app/controllers/concerns/lti.rb

```diff
@@ -226,6 +226,7 @@ def store_lti_session_data(parameters)
     @lti_parameters.save!
 
     session[:external_user_id] = current_user.id
+    session[:pair_programming] = params[:custom_pair_programming] || false
```

Comment:
Where are we removing this variable again? At some point, we should probably do so.